app/controllers/posts_controller.rb

```diff
@@ -28,6 +28,17 @@ def show
     @post = Post.find(params[:id])
   end
 
```

Comment:
Kindly, use `CanCanCan` for Post authorization here in `Post Controller`. To do that you can include something like this at the top between lines 1 and 2: 
```
load_and_authorize_resource
```
You can read more [here](https://www.rubydoc.info/github/CanCanCommunity/cancancan/CanCan%2FControllerAdditions%2FClassMethods:load_and_authorize_resource).

Reply:
I have added this. Thank you @Mre55 ☺️